lib/src/testCommon/kotlin/org/didcommx/didcomm/mock/AliceNewSecretResolverMock.kt

```diff
@@ -0,0 +1,160 @@
+package org.didcommx.didcomm.mock
+
+import org.didcommx.didcomm.common.VerificationMaterial
+import org.didcommx.didcomm.common.VerificationMaterialFormat
+import org.didcommx.didcomm.common.VerificationMethodType
+import org.didcommx.didcomm.secret.Secret
+import org.didcommx.didcomm.secret.SecretResolverInMemory
+import java.util.Optional
+
+class AliceNewSecretResolverMock : SecretResolverInMemoryMock {
```

Comment:
What is a difference between AliceNewSecretResolverMock and AliceSecretResolverMock?